manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: simonmichael.hledger
+PackageVersion: 1.29.2
+InstallerType: zip
+FileExtensions:
+- exe
+ReleaseDate: 2023-04-08
```

```suggestion
ReleaseDate: 2023-04-07
```
According to https://hledger.org/release-notes.html#hledger-1-29-2